### ISSUE TITLE
Run single rule

### ISSUE THREAD

**Issue by stefanbuck** [eslint/eslint#6333]
While moving a bigger project from jscs to eslint, I found myself wanting to have the possibility to run a specific or subset of rules instead of full eslint config from the project. 

In addition this should also work for the auto fix command. This would allow to split up the auto fix changes into logical commits.

**Comment by vitorbal** [comment]
Hi, @stefanbuck, thank you for your suggestion! Could you go into a bit more detail about what this option would look like, how would it be run from the cli, etc?

You mention wanting to run a subset of rules from an eslint config. Are you using a shared config to run eslint in your project? If so, how would this option you suggest be different than just disabling the rules you're not interested in, in your .eslintrc file?

**Comment by eslintbot** [comment]
Hi @stefanbuck, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by alberto** [comment]
There is [eslint-nibble](https://github.com/IanVS/eslint-nibble) by @IanVS, which I think does exactly that.

**Comment by platinumazure** [comment]
You can also do this with a CLI invocation like the following:

```bash
eslint --no-eslintrc --rule some-rule:error
```

**Comment by stefanbuck** [comment]
`eslint --no-eslintrc --rule` that's the solution that I'm looking for. Thanks @platinumazure 

@alberto will also look into `eslint-nibble`, looks promising.
